extension/encoding/awslogsencodingextension/README.md

```diff
@@ -128,3 +128,43 @@ extensions:
+| `httpSourceId`                | `aws.waf.source.id`                                                                              |
+| `httpRequest.clientIp`        | `client.address`                                                                                 |
+| `httpRequest.headers`         | Each header is saved under `http.request.header.<header_name>`                                   |
+| `httpRequest.uri`             | `url.path`                                                                                       |
+| `httpRequest.args`            | `url.query`                                                                                      |
+| `httpRequest.httpMethod`      | `http.request.method`                                                                            |
+| `httpRequest.httpVersion`     | Splits in:<br>1.`network.protocol.name`<br>2.`network.protocol.version`                          |
+| `httpRequest.requestId`       | `aws.request_id`                                                                                 |
+| `httpRequest.fragment`        | `url.fragment`                                                                                   |
+| `httpRequest.scheme`          | `url.scheme`                                                                                     |
+| `httpRequest.country`         | `geo.country.iso_code`                                                                           |
+| `httpRequest.host`            | `server.address`                                                                                 |
+| `responseCodeSent`            | `http.response.status_code`                                                                      |
+| `ja3Fingerprint`              | `tls.client.ja3`                                                                                 |
+| `ja4Fingerprint`              | `tls.client.ja4`                                                                                 |
+| `formatVersion`               | _Currently not supported_                                                                        |
+| `terminatingRuleMatchDetails` | _Currently not supported_                                                                        |
+| `ruleGroupList`               | _Currently not supported_                                                                        |
+| `rateBasedRuleList`           | _Currently not supported_                                                                        |
+| `nonTerminatingMatchingRules` | _Currently not supported_                                                                        |
+| `requestHeadersInserted`      | _Currently not supported_                                                                        |
+| `labels`                      | _Currently not supported_                                                                        |
```

Comment:
nit - labels might be useful when evaluating logs. And parsing should be straightforward given they have "string":"string" format (ignoring character restrictions) [1]

[1] - https://docs.aws.amazon.com/waf/latest/developerguide/waf-rule-label-requirements.html